### DISCUSSION TITLE
[Bug] The binary files installed with extras/Reaper have changed, the manifest is now wrong

### DISCUSSION THREAD

**Discussion by dna1000** [ScoopInstaller/Scoop#5710] [Help and Questions]
Hi

I'm not savvy enough to work out how to use github to make a change to the **reaper.json** manifest from the **extras** bucket.
The new version (v7) of Reaper has removed the "reamote.exe" application from the files installed but the manifest still references it, so the install fails (although it does install and work but keeps wanting to update as it doesn't finish cleanly).

Snippet from the manifest (original)

>     .......
>     "bin": [
>         "reaper.exe",
>         "reamote.exe"
>     ],
>     "shortcuts": [
>         [
>             "reaper.exe",
>             "Reaper"
>         ],
>         [
>             "reamote.exe",
>             "ReaMote"
>         ]
>     ],
>     .......

Needs to change to 

>      .......
>     "bin": [
>         "reaper.exe"
>     ],
>     "shortcuts": [
>         [
>             "reaper.exe",
>             "Reaper"
>         ]
>     ],
>      .......

I think this is the change required, I've overwritten my local manifest and it installs cleanly (but then breaks the extras bucket when I next update). I can't work out how to do the github stuff to make this change (or even how to go about getting permission to do so).

Just posting in case someone who knows how to do github stuff could update sometime.

Dave

**Comment by HUMORCE** [comment]
You should create issue in Extras bucket.

https://github.com/ScoopInstaller/Extras/pull/12163
